Cargo.toml

```diff
@@ -132,6 +132,17 @@ jj-lib = { path = "lib", version = "0.22.0" }
 jj-lib-proc-macros = { path = "lib/proc-macros", version = "0.22.0" }
 testutils = { path = "lib/testutils" }
 
+[workspace.lints.clippy]
+all = "warn"
+explicit_iter_loop = "warn"
+flat_map_option = "warn"
+implicit_clone = "warn"
+needless_for_each = "warn"
+semicolon_if_nothing_returned = "warn"
+# Until all documentation comments are fixed, do not warn about them
+too_long_first_doc_paragraph = { level = "allow", priority = 1 }
```

Comment:
got unknown lint warning on stable.

Reply:
Darn. Added a temporary `allow` for unknown lint names. That should take care of it.

Reply:
If there aren't many places to annotate, it might be better to add scoped `#[allow(too_long_first_doc_paragprah)] #[allow(unknown_lints)]`.

Reply:
In fact, adding summaries (for a better documentation) will be even better: #4581